apache-Dockerfile-block-1

```diff
@@ -5,7 +5,7 @@ RUN rm -rf /var/www/html && mkdir -p /var/lock/apache2 /var/run/apache2 /var/log
 # Apache + PHP requires preforking Apache for best results
 RUN a2dismod mpm_event && a2enmod mpm_prefork
 
-RUN mv /etc/apache2/apache2.conf /etc/apache2/apache2.conf.dist
+RUN mv /etc/apache2/apache2.conf /etc/apache2/apache2.conf.dist && rm /etc/apache2/{conf,sites}-enabled/*
```

Comment:
Ok, fair enough; just know that I'll be right back to say "I told you so!" if I ever find myself needing to refer to these. :smile: :+1:

So, here's my new problem with this line (a technical one instead of a philosophical one):
```console
$ docker build -
FROM debian
RUN echo {a,b,c}d
^D
Sending build context to Docker daemon 2.048 kB
Sending build context to Docker daemon 
Step 0 : FROM debian
 ---> 1265e16d0c28
Step 1 : RUN echo {a,b,c}d
 ---> Running in 190548a111be
{a,b,c}d
 ---> c749377d911b
Removing intermediate container 190548a111be
Successfully built c749377d911b
```

`RUN` uses `/bin/sh` not `/bin/bash`, so `{...,...}` syntax doesn't work here. :+1: